<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CompositeTileSourceWidgetClass</class>
 <widget class="QWidget" name="CompositeTileSourceWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CompositeTileSourceWidget</string>
  </property>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>39</y>
     <width>375</width>
     <height>295</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="dragDropMode">
    <enum>QAbstractItemView::InternalMove</enum>
   </property>
   <property name="uniformItemSizes">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>248</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QToolButton" name="addSourceButton">
      <property name="text">
       <string>+</string>
      </property>
      <property name="icon">
       <iconset resource="../../../MapGraphics-master/MapGraphics/resources.qrc">
        <normaloff>:/icons/images/edit_add.png</normaloff>:/icons/images/edit_add.png</iconset>
      </property>
      <property name="popupMode">
       <enum>QToolButton::InstantPopup</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="removeSourceButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>X</string>
      </property>
      <property name="icon">
       <iconset resource="../../../MapGraphics-master/MapGraphics/resources.qrc">
        <normaloff>:/icons/images/editdelete.png</normaloff>:/icons/images/editdelete.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="moveDownButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>D</string>
      </property>
      <property name="arrowType">
       <enum>Qt::DownArrow</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="moveUpButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>U</string>
      </property>
      <property name="arrowType">
       <enum>Qt::UpArrow</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="opacitySlider">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="tracking">
       <bool>false</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../../MapGraphics-master/MapGraphics/resources.qrc"/>
 </resources>
 <connections/>
</ui>
